ivy/functional/backends/numpy/random.py

```diff
@@ -29,7 +28,6 @@ def random_normal(
     shape: Optional[Union[ivy.NativeShape, Sequence[int]]] = None,
     *,
     device: str,
-    out: Optional[np.ndarray] = None,
 ) -> np.ndarray:
```

Comment:
with the new policy, we should not be deleted `out` arguments from any backends, even if unused